publish.sh

```diff
@@ -1,3 +1,3 @@
 rm -rf dist/*
-python setup.py sdist
+python -m build --sdist
```

Comment:
this change is in accordance with:
https://packaging.python.org/en/latest/discussions/setup-py-deprecated/#what-commands-should-be-used-instead